### PR TITLE
Multi-site docs update (BSC#1144717)

### PR DESCRIPTION
**Pull request by asettle**
Updating multi-site docs and adding new section
for updating the dashboard.

### CODE REVIEW

File: xml/admin_ceph_gateway.xml [hunk 1]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
 rgw_zone=us-east-2</screen>
 <screen>&prompt.cephuser;sudo systemctl start ceph-radosgw@rgw.us-east-2</screen>
    </sect3>
-  </sect2>
+ <sect3 xml:id="ceph-update-dashboard-config">
+   <title>Update Dashboard Configuration</title>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
   <title>Update &dashboard; Configuration</title>
```

File: xml/admin_ceph_gateway.xml [hunk 2]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
...
+   <procedure>
+     <step>
+       <para>Set the following to update your dashboard configuration for
+       multi-site:</para>
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key ACCESS-KEY
+       &prompt.cephuser; ceph dashboard set-rgw-api-secret-key SECRET-KEY
+      </screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: Your indentation in this case causes the second prompt to be indented. screen markup should be sort of pushed to the left. Run `daps xmlformat` and check its result before comitting. It's useful for unifying the diffs.

```
<screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key <replaceable>ACCESS-KEY</replaceable>
&prompt.cephuser; ceph dashboard set-rgw-api-secret-key <replaceable>SECRET-KEY</replaceable>
</screen>
```

File: xml/admin_ceph_gateway.xml [hunk 3]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
...
+      <para>Based on the secret keys and access key variables.</para>
+     </step>
+     <step>
+       <para>Set the dashboard default to <literal>admin</literal>:</para>
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id USER-ID</screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: We want to deaful to the 'admin' user, right? Then i would supply the 'admin' USER-ID:
```suggestion
       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id admin</screen>
```

File: xml/admin_ceph_gateway.xml [hunk 4]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
...
+     </step>
+     <step>
+       <para>Disable and re-eanble the dashboard to apply settings.</para>
+       <screen>&prompt.cephuser; ceph mgr module disable dashboard
+       &prompt.cephuser; ceph mgr module enable dashboard</screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: The indenting stuff again, it's going to look stairs-like when rendered.

File: xml/admin_ceph_gateway.xml [hunk 5]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
...
+     </step>
+     <step>
+       <para>If the RGW node changed or a load balancer is used instead of
+         the RGWs, update the dashboard:</para>
+       <screen>&prompt.cephuser; dashboard set-rgw-api-host HOST
+       &prompt.cephuser; dashboard set-rgw-api-port PORT</screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: Indentation.

File: xml/admin_ceph_gateway.xml [hunk 6]
```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
...
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id USER-ID</screen>
+     </step>
+     <step>
+       <para>Disable and re-eanble the dashboard to apply settings.</para>
+       <screen>&prompt.cephuser; ceph mgr module disable dashboard
+       &prompt.cephuser; ceph mgr module enable dashboard</screen>
+     </step>
+     <step>
+       <para>If the RGW node changed or a load balancer is used instead of
+         the RGWs, update the dashboard:</para>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]: Within the SES team, we agreed on replacing the upstream term RGW (meaning RADOS gateway) with OGW (Object gateway) throughout the docs. And there is an entity for it as well: &ogw;

**Comment by tbazant** [comment]
@asettle , apart from our peer review, i would ask the bug owner for technical review here probably. I've never tested multisite OGW's myself.

File: xml/admin_ceph_gateway.xml (hunk 2 above)
Comment (asettle) [reply xml/admin_ceph_gateway.xml]: Good tip! Did'nt know that one :)

**Comment by asettle** [comment]
> @asettle , apart from our peer review, i would ask the bug owner for technical review here probably. I've never tested multisite OGW's myself.

I've requested Alejandro (bug reporter) to review, I'll add a NEEDINFO flag to the BZ.

File: xml/admin_ceph_gateway.xml [hunk 7]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
 <screen>&prompt.cephuser;sudo systemctl start ceph-radosgw@rgw.us-east-2</screen>
    </sect3>
-  </sect2>
+ <sect3 xml:id="ceph-update-dashboard-config">
+   <title>Update &dashboard; Configuration</title>
+   <procedure>
+     <step>
+       <para>Set the following to update your dashboard configuration for
+       multi-site:</para>
+<screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key <replaceable>ACCESS-KEY</replaceable>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
<screen>&prompt.cephuser;ceph dashboard set-rgw-api-access-key <replaceable>ACCESS-KEY</replaceable>
```

File: xml/admin_ceph_gateway.xml [hunk 8]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+&prompt.cephuser; ceph dashboard set-rgw-api-secret-key <replaceable>SECRET-KEY</replaceable>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
&prompt.cephuser;ceph dashboard set-rgw-api-secret-key <replaceable>SECRET-KEY</replaceable>
```

File: xml/admin_ceph_gateway.xml [hunk 9]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+</screen>
+      <para>Based on the secret keys and access key variables.</para>
+     </step>
+     <step>
+       <para>Set the dashboard default to <literal>admin</literal>:</para>
+<screen>&prompt.cephuser; ceph dashboard set-rgw-api-user-id admin</screen>
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
<screen>&prompt.cephuser;ceph dashboard set-rgw-api-user-id admin</screen>
```

File: xml/admin_ceph_gateway.xml [hunk 10]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+     </step>
+     <step>
+       <para>Disable and re-eanble the dashboard to apply settings.</para>
+<screen>&prompt.cephuser; ceph mgr module disable dashboard
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
<screen>&prompt.cephuser;ceph mgr module disable dashboard
```

File: xml/admin_ceph_gateway.xml [hunk 11]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+&prompt.cephuser; ceph mgr module enable dashboard
+</screen>
+     </step>
+     <step>
+       <para>If the &ogw; node changed or a load balancer is used instead of
+         the &ogw;s, update the dashboard:</para>
+<screen>&prompt.cephuser; dashboard set-rgw-api-host HOST
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
<screen>&prompt.cephuser;dashboard set-rgw-api-host HOST
```

File: xml/admin_ceph_gateway.xml [hunk 12]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+&prompt.cephuser; dashboard set-rgw-api-port PORT
```
Comment (tbazant) [inline xml/admin_ceph_gateway.xml]:
```suggestion
&prompt.cephuser;dashboard set-rgw-api-port PORT
```

File: xml/admin_ceph_gateway.xml [hunk 13]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+     <step>
+       <para>Disable and re-eanble the dashboard to apply settings.</para>
+<screen>&prompt.cephuser;ceph mgr module disable dashboard
+&prompt.cephuser; ceph mgr module enable dashboard
+</screen>
+     </step>
+     <step>
+       <para>If the &ogw; node changed or a load balancer is used instead of
+         the &ogw;s, update the dashboard:</para>
+<screen>&prompt.cephuser;dashboard set-rgw-api-host HOST
```
Comment (votdev) [inline xml/admin_ceph_gateway.xml]: I'm not familiar with this XML format, but i think this should be
```suggestion
<screen>&prompt.cephuser;dashboard set-rgw-api-host <replaceable>HOST</replaceable>
```

File: xml/admin_ceph_gateway.xml [hunk 14]
```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
...
+&prompt.cephuser;dashboard set-rgw-api-port PORT
```
Comment (votdev) [inline xml/admin_ceph_gateway.xml]:
```suggestion
&prompt.cephuser;dashboard set-rgw-api-port <replaceable>PORT</replaceable>
```

File: xml/admin_ceph_gateway.xml (hunk 13 above)
Comment (asettle) [reply xml/admin_ceph_gateway.xml]: Correct! Thanks @votdev :)

**Comment by asettle** [comment]
@votdev hopefully that fixes everything for you :) let me know if it's good to go

**Pull request merged**